docs/getting-started-bankid.md

```diff
@@ -211,11 +211,14 @@ BankId options allows you to set and override some options such as these.
     // Limit possible login methods to, for example, only allow BankID on smartcard.
     options.BankIdCertificatePolicies = BankIdCertificatePolicies.GetPoliciesForProductionEnvironment(...);
 
-	// Issue birthdate claim based on data extracted from the personal identity number
```

Comment:
The following lines had white space issues where tabs were not presented in the same way as spaces in the markdown-render.